app/controllers/application_controller.rb

```diff
@@ -1,3 +1,32 @@
 class ApplicationController < ActionController::Base
-  include SessionsHelper
+  def sign_in(user)
+    token = generate_token
+    user.update_attribute(:remember_token, token)
+    cookies.signed.permanent[:remember_token] = token
+    set_current_user(user)
+  end
+
+  def sign_in?
+    !current_user.nil?
```

Comment:
Not really, we don't use it unless a user is currently signed in. and since we already have a handler for that, it doesn't throw an undefined method error.